aquacomputer_d5next.c

```diff
@@ -765,14 +1204,50 @@ static int aqc_probe(struct hid_device *hdev, const struct hid_device_id *id)
 		ret = -ENOMEM;
 		goto fail_and_close;
 	}
-
+	
 	mutex_init(&priv->mutex);
 
+	if (priv->num_fans > 0) {
+		priv->groups[1] = &aqc_curve_pt_1_pwm_group;
```

Comment:
The groups should be filled starting from 0. In the current state groups[0] is NULL leading to everything after that being ignored

Reply:
fixed in f65a7cd3!